packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -163,6 +163,62 @@ function bindEvents(dispatch, call) {
   call.on('error', (err) => dispatch(updateCallError(call, err)));
 }
 
+function handleIncomingCalls(sparkInstance, dispatch) {
```

Comment:
The passing of dispatch always felt weird to me... I've generally turned things like this into thunks, but I can't really come up with a good reason besides "consistency"